src/client.rs

```diff
@@ -1079,15 +1079,28 @@ impl Spotify {
     ///
     /// Parameters:
     /// - limit - the number of entities to return
+    /// - time_limit - A Unix timestamp in milliseconds.
     ///
     /// [Reference](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-the-users-currently-playing-track)
     #[maybe_async]
-    pub async fn current_user_recently_played<L: Into<Option<u32>>>(
+    pub async fn current_user_recently_played<L: Into<Option<u32>>, T: Into<Option<TimeLimits>>>(
         &self,
         limit: L,
+        time_limit: T,
     ) -> ClientResult<CursorBasedPage<PlayHistory>> {
-        let mut params = Query::with_capacity(1);
+        let mut params = Query::with_capacity(2);
         params.insert("limit".to_owned(), limit.into().unwrap_or(50).to_string());
+        if let Some(l) = time_limit.into() {
+            match l {
```

Comment:
Perhaps you could give these variables a more descriptive name than `l` and `y`, it's a little bit meaningless.